src/App.css

```diff
@@ -1,39 +1,49 @@
 .App {
-  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.calculator {
+  width: 80%;
 }
```

Comment:
- [ ] In order to match the required calculator design, I suggest you add a `max-width` property to the calculator class

| Required Design            | Your Design   |
| ------------------- | -------------- |
| ![image](https://user-images.githubusercontent.com/60772732/133642949-f06da818-1c30-4b8b-84ed-4748302e3c6b.png)  | ![image](https://user-images.githubusercontent.com/60772732/133642826-9223f3e7-06b8-4888-a0bb-dff7f370c3a4.png)    |

Reply:
Done this!